src/ComponentMap.js

```diff
@@ -0,0 +1,13 @@
+
+export function find(paymentMethods, type) {
+    return paymentMethods.paymentMethods.find(pm => pm.type === mapCreatedComponentType(type));
+}
+
+function mapCreatedComponentType(pmType) {
+    // Components created as 'card' need to be matched with paymentMethod response objects with type 'scheme'
+    return pmType === 'card' ? 'scheme' : pmType;
+}
+
+export const nativeComponents = [
```

Comment:
Usually constants are uppercased. Most of devs follow [this rule](https://github.com/airbnb/javascript#naming--uppercase)

Reply:
👍